e2e-tests/src/test/utility.rs

```diff
@@ -0,0 +1,53 @@
+use log::info;
+
+use substrate_api_client::{compose_call, compose_extrinsic, GenericAddress, XtStatus};
+use sp_core::Pair;
+use crate::transfer::{setup_for_transfer};
+use std::fmt::Write;
+
+use codec::{Compact};
+
+use crate::config::Config;
+
+pub fn batch_transactions(config: Config) -> anyhow::Result<()> {
+    const NUMBER_OF_TRANSACTIONS : usize = 100;
+
+    let (connection, _, to) = setup_for_transfer(config);
+
+    let call = compose_call!(
+        connection.metadata,
+        "Balances",
+        "transfer",
+        GenericAddress::Id(to.clone()),
+        Compact(1000u128)
+    );
+    let mut transactions = Vec::new();
+    for _i in 0..NUMBER_OF_TRANSACTIONS {
+        transactions.push(call.clone());
+    }
+
+    let extrinsic = compose_extrinsic!(
+        connection,
+        "Utility",
+        "batch",
+        transactions
+    );
+
+    let finalized_block_hash = connection
+        .send_extrinsic(extrinsic.hex_encode(), XtStatus::Finalized)
+        .expect("Could not send extrinsc")
+        .expect("Could not get tx hash");
+    info!("[+] A batch of {} transactions was included in {} block.", NUMBER_OF_TRANSACTIONS, finalized_block_hash);
+
+    let block_with_batch_tx = connection.get_block::<aleph_runtime::Block>(Some(finalized_block_hash)).
```

Comment:
YMMV, but not sure if it's worth pulling WASM, and entire runtime just to have the Block. There should be some generic Block definition in `sp_runtime::{generic}` ?

Reply:
Yeah, this is heavy I agree to add it. Let me explore one more time if I can find some other solution which does not involve including whole runtime.

Reply:
I've reverted dependency on `aleph-runtime` in `e2e-tests` package. Decision was made on the fact build of e2e workflow varied from 45 to 50 mins, not exactly sure it was due to that change but still a long time. Test is good enough since `send_extrinsic` is a blocking call, and it waits for a finalized block in chain with that transaction.